libr/core/cmd_eval.c

```diff
@@ -30,26 +30,36 @@ static const char *help_msg_e[] = {
 
 static const char *help_msg_ec[] = {
 	"Usage ec[s?] [key][[=| ]fg] [bg]", "", "",
-	"ec", "", "list all color keys",
+	"ec", " [key]", "list all/key color keys",
 	"ec*", "", "same as above, but using r2 commands",
 	"ecd", "", "set default palette",
 	"ecr", "", "set random palette (see also scr.randpal)",
 	"ecs", "", "show a colorful palette",
 	"ecj", "", "show palette in JSON",
 	"ecc", " [prefix]", "show palette in CSS",
-	"eco", " dark|white", "load white color scheme template",
+	"eco", " [theme]", "load theme if provided (list available themes if not)",
 	"ecp", "", "load previous color theme",
 	"ecn", "", "load next color theme",
-	"ecH", "[?]", "highlight word or instruction",
+	"ecH", " [?]", "highlight word or instruction",
 	"ec", " prompt red", "change color of prompt",
 	"ec", " prompt red blue", "change color and background of prompt",
-	"", " ", "",
+	"Vars:", "", "",
 	"colors:", "", "rgb:000, red, green, blue, #ff0000, ...",
 	"e scr.color", "=0", "use more colors (0: no color 1: ansi 16, 2: 256, 3: 16M)",
 	"$DATADIR/radare2/cons", "", R_JOIN_2_PATHS ("~", R2_HOME_THEMES) " ./",
 	NULL
 };
 
+static const char *help_msg_eco[] = {
+	"Usage: eco [theme]", "", "load theme (cf. Path and dir.prefix)",
```

Comment:
What about `	"Usage: eco[jc] [theme]", "", "load theme (cf. Path and dir.prefix)",` instead?

Reply:
Definitely.